src/extractor/ros2/main.py

```diff
@@ -1,37 +1,34 @@
-
-#import ros1_extract
 import sys
-from datetime import datetime
 from rosbags.rosbag2 import Reader
 import ros2_extract
 
-def date_to_datetime(time):
-    return datetime.strptime(time, "%Y-%m-%d %H:%M:%S")
-
 
 def main():
     path_to_file = sys.argv[1]
 
+    # time parsed as in second (str)
     start = sys.argv[2]
     end = sys.argv[3]
 
     with Reader(path_to_file) as reader:
-        bag_start = datetime.fromtimestamp(reader.start_time // 1000000000)
-        bag_end = datetime.fromtimestamp(reader.end_time // 1000000000)
-        # print("The bagfile STARTS at：", bag_start, " and ENDS at: ", bag_end)
+        # time in second
+        bag_start = reader.start_time / 1000000000
+        bag_end = reader.end_time / 1000000000
+
+        # bag_duration = reader.duration / 1000000000
 
     if start == 'start':
         start_t = bag_start
     else:
         try:
-            if date_to_datetime(start) < bag_start or date_to_datetime(start) > bag_end:
+            if float(start) < bag_start or float(start) > bag_end:
                 print("Please input a valid start time.")
```

Comment:
If I call

 ./extractor.sh ros2 1 3 bagfiles/ros2/simple/

it returns "Please input a valid start time". It should convert the 1 to the second 1 of the file. The same goes for the second argument.